public class solid_Rhombus {

    public static void Solid_Rhombus(int n){

        //outer
        for(int i=1; i<=n; i++){
            //inner
            for(int j=1; j<=n-i; j++){
                System.out.print(" "+" ");
            }
            for(int j=1; j<=n; j++){
                System.out.print("*"+" ");
            }
            System.out.println();
        }

    }

    public static void main(String[] args){
        Solid_Rhombus(5);
    }
    
}
